samples/snippets/create_lite_pubsub_export_subscription_example.py

```diff
@@ -0,0 +1,142 @@
+
+"""This application demonstrates how to create a subscription that writes
+messages to a Pub/Sub topic. For more information, see documentation at
+https://cloud.google.com/pubsub/lite/docs/export-subscriptions.
+"""
+
+import argparse
+
+
+def create_lite_pubsub_export_subscription(
+    project_number,
+    cloud_region,
+    zone_id,
+    topic_id,
+    subscription_id,
+    pubsub_topic_id,
+    regional,
+    target_location,
+):
+    # [START pubsublite_create_pubsub_export_subscription]
+    from google.api_core.exceptions import AlreadyExists
+    from google.cloud.pubsub_v1 import PublisherClient
+    from google.cloud.pubsublite import AdminClient, Subscription, ExportConfig
+    from google.cloud.pubsublite.types import (
+        CloudRegion,
+        CloudZone,
+        SubscriptionPath,
+        TopicPath,
+    )
+
+    # TODO(developer):
+    # project_number = 1122334455
+    # cloud_region = "us-central1"
+    # zone_id = "a"
+    # topic_id = "your-topic-id"
+    # subscription_id = "your-subscription-id"
+    # pubsub_topic_id = "destination-topic-id"
+    # regional = True
+    # target_location = BacklogLocation.BEGINNING
+
+    if regional:
+        location = CloudRegion(cloud_region)
+    else:
+        location = CloudZone(CloudRegion(cloud_region), zone_id)
+
+    topic_path = TopicPath(project_number, location, topic_id)
+    subscription_path = SubscriptionPath(project_number, location, subscription_id)
+    destination_topic_path = PublisherClient.topic_path(project_number, pubsub_topic_id)
+
+    subscription = Subscription(
+        name=str(subscription_path),
+        topic=str(topic_path),
+        delivery_config=Subscription.DeliveryConfig(
+            # Possible values for delivery_requirement:
+            # - `DELIVER_IMMEDIATELY`
+            # - `DELIVER_AFTER_STORED`
+            # You may choose whether to wait for a published message to be successfully written
+            # to storage before the server delivers it to subscribers. `DELIVER_IMMEDIATELY` is
+            # suitable for applications that need higher throughput.
+            delivery_requirement=Subscription.DeliveryConfig.DeliveryRequirement.DELIVER_IMMEDIATELY,
+        ),
+        export_config=ExportConfig(
+            # Possible values for desired_state:
+            # - `ACTIVE`: enable message processing.
+            # - `PAUSED`: suspend message processing.
+            desired_state=ExportConfig.State.ACTIVE,
+            pubsub_config=ExportConfig.PubSubConfig(
+                topic=destination_topic_path,
+            ),
+        ),
+    )
+
+    client = AdminClient(cloud_region)
+    try:
+        response = client.create_subscription(subscription, target_location)
+        print(f"{response.name} created successfully.")
+    except AlreadyExists:
+        print(f"{subscription_path} already exists.")
+    # [END pubsublite_create_pubsub_export_subscription]
+
+
+if __name__ == "__main__":
```

Comment:
We should avoid CLIs: https://googlecloudplatform.github.io/samples-style-guide/#no-cli

Reply:
I've removed the CLI